### PR TITLE
Initial add of ASP.NET 2.1.0 templates

### PR DESCRIPTION
**Pull request by mlorbetske**
Apparently this picks up 4 of the line ending changes for vbproj files as well

### CODE REVIEW

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/EmptyWeb-FSharp/.template.config/template.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+  "author": "Microsoft",
+  "classifications": ["Web", "Empty"],
+  "name": "ASP.NET Core Empty",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An empty project template for creating an ASP.NET Core application. This template does not have any content in it.",
+  "groupIdentity": "Microsoft.Web.Empty",
+  "precedence": "2000",
```
Comment (seancpeters) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/EmptyWeb-FSharp/.template.config/template.json]: Precedence 3000

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/RazorPagesWeb-CSharp/.template.config/template.json [hunk 2]
```diff
@@ -0,0 +1,363 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Web", "MVC", "Razor Pages" ],
+  "name": "ASP.NET Core Web App",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A project template for creating an ASP.NET Core application with example ASP.NET Razor Pages content",
+  "groupIdentity": "Microsoft.Web.RazorPages",
+  "precedence": "2000",
```
Comment (seancpeters) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/RazorPagesWeb-CSharp/.template.config/template.json]: Precedence 3000

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/StarterWeb-CSharp/.template.config/template.json [hunk 3]
```diff
@@ -0,0 +1,357 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": ["Web", "MVC"],
+  "name": "ASP.NET Core Web App (Model-View-Controller)",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A project template for creating an ASP.NET Core application with example ASP.NET Core MVC Views and Controllers. This template can also be used for RESTful HTTP services.",
+  "groupIdentity": "Microsoft.Web.Mvc",
+  "precedence": "2000",
```
Comment (seancpeters) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/StarterWeb-CSharp/.template.config/template.json]: Precedence 3000

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/StarterWeb-FSharp/.template.config/template.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": ["Web", "MVC"],
+  "name": "ASP.NET Core Web App (Model-View-Controller)",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A project template for creating an ASP.NET Core application with example ASP.NET Core MVC Views and Controllers. This template can also be used for RESTful HTTP services.",
+  "groupIdentity": "Microsoft.Web.Mvc",
+  "precedence": "2000",
```
Comment (seancpeters) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/StarterWeb-FSharp/.template.config/template.json]: Precedence 3000

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/WebApi-CSharp/.template.config/template.json [hunk 5]
```diff
@@ -0,0 +1,278 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": ["Web", "WebAPI"],
+  "name": "ASP.NET Core Web API",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A project template for creating an ASP.NET Core application with an example Controller for a RESTful HTTP service. This template can also be used for ASP.NET Core MVC Views and Controllers.",
+  "groupIdentity": "Microsoft.Web.WebApi",
+  "precedence": "2000",
```
Comment (seancpeters) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/WebApi-CSharp/.template.config/template.json]: Precedence 3000

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/WebApi-FSharp/.template.config/template.json [hunk 6]
```diff
@@ -0,0 +1,67 @@
+{
+  "author": "Microsoft",
+  "classifications": ["Web", "WebAPI"],
+  "name": "ASP.NET Core Web API",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A project template for creating an ASP.NET Core application with an example Controller for a RESTful HTTP service. This template can also be used for ASP.NET Core MVC Views and Controllers.",
+  "groupIdentity": "Microsoft.Web.WebApi",
+  "precedence": "2000",
```
Comment (seancpeters) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/WebApi-FSharp/.template.config/template.json]: Precedence 3000

**Comment by mlorbetske** [comment]
Merging this as I think it's right for now

**Pull request merged**